blazorbootstrap/Components/Button/Button.razor.cs

```diff
@@ -201,9 +201,6 @@ private void SetAttributes()
             if (!Attributes.TryGetValue("role", out _))
                 Attributes.Add("role", "button");
 
-            if (!Attributes.TryGetValue("href", out _))
-                Attributes.Add("href", To!);
-
```

Comment:
@MarvinKlein1508 Instead of changing the markup, please add an else statement for consistency.